deid/dicom/filter.py

```diff
@@ -26,37 +26,27 @@ def apply_filter(dicom, field, filter_name, value):
     dicom: the pydicom.dataset Dataset (pydicom.read_file)
     field: the name of the field to apply the filter to,
       or the tag number as a string '0xGGGGEEEE'
-    filer_name: the name of the filter to apply (e.g., contains)
+    filter_name: the name of the filter to apply (e.g., contains)
     value: the value to set, if filter_name is valid
 
     """
     if "0x" in field:
         field = int(field, 0)  # 0=decode hex with 0x prefix
     filter_name = filter_name.lower().strip()
+    filter_name_map = {
```

Comment:
I see what you are trying to do, but this design isn't an improvement because in creating the dictionary you are executing each function. We definitely don't want to do that.

Reply:
Thank you for the feedback.

Reply:
If you’d like to revert this change, the docstring fixes are great and would be much appreciated!